selenium_webdriver/src/test/java/tests/sauceDemoTests/SauceDemoDeleteItemFromCartTest.java

```diff
@@ -2,24 +2,24 @@
 
 import model.User;
 import org.testng.annotations.Test;
-import pageobject.sauceDemoPages.SauceDemoLoginPage;
-import pageobject.sauceDemoPages.SauceDemoMainPage;
+import pageobject.SauceDemoLoginPage;
 import service.UserCreator;
 import tests.BaseTest;
 
-import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertEquals;
 
 public class SauceDemoDeleteItemFromCartTest extends BaseTest {
 
     @Test
-    public void oneCanDeleteItemFromCart() {
-        User testUser = UserCreator.withCredentialsFromProperty();
-        SauceDemoMainPage itemWasDeletedFromCart = new SauceDemoLoginPage(driver)
+    public void oneCanDeleteItemFromCartCheck() {
+        User testUser = UserCreator.getStandardUser();
+        String itemWasDeletedFromCart = new SauceDemoLoginPage(driver)
```

Comment:
incorrect variable name. Here you deal with button/element/locator Text, thus you need to use buttonText or even text

Reply:
Fixed